GUI/SearchTaskWindow.xaml

```diff
@@ -309,7 +557,17 @@
                                     </TreeView>
                                 </StackPanel>
                                 <StackPanel Grid.Column="1">
-                                    <Label Content="Variable" />
+                                    <Label Content="Variable"  ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                        <Label.ToolTip>
+                                            <TextBlock>
+                                                Only use this for highly prevelant modifications (>10% of peptides).
+                                                <LineBreak/>
+                                                All PTMs that are annotated in the database (e.g. with G-PTM-D) will be automatically searched.
```

Comment:
I still think this is confusing to newbies and needs to be more explicit:

All PTMs that are annotated in the database (e.g. by the G-PTM-D task) will be automatically searched even without these boxes checked.